### DISCUSSION TITLE
Welcome to Skul.Mod Discussions!

### DISCUSSION THREAD

**Discussion by Tobi-Mob** [Tobi-Mob/Skul.Mod#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by lazar555** [comment]
can you tell me when will the mod be updated to v1.7.0?

**Reply by Tobi-Mob** [reply]
Hi,

I put something together: https://github.com/Tobi-Mob/Skul.Mod/releases/tag/v1.0.3
The mod starts on my machine and the autoattack works. But I did not test any further.

So I can not guarantee that everything works, but I hope for the best.
